casper/src/main/scala/coop/rchain/casper/util/comm/CasperPacketHandlerInternal.scala

```diff
@@ -0,0 +1,371 @@
+package coop.rchain.casper.util.comm
+
+import cats.data.EitherT
+import cats.effect.concurrent.Ref
+import cats.effect.{Concurrent, Sync}
+import cats.implicits._
+import cats.{Applicative, Monad}
+import com.google.protobuf.ByteString
+import coop.rchain.blockstorage.{BlockDagStorage, BlockStore}
+import coop.rchain.casper.Estimator.Validator
+import coop.rchain.casper.LastApprovedBlock.LastApprovedBlock
+import coop.rchain.casper.MultiParentCasperRef.MultiParentCasperRef
+import coop.rchain.casper._
+import coop.rchain.casper.genesis.Genesis
+import coop.rchain.casper.protocol._
+import coop.rchain.casper.util.rholang.RuntimeManager
+import coop.rchain.catscontrib.Catscontrib._
+import coop.rchain.catscontrib.MonadTrans
+import coop.rchain.comm.CommError.ErrorHandler
+import coop.rchain.comm.discovery.NodeDiscovery
+import coop.rchain.comm.protocol.routing.Packet
+import coop.rchain.comm.rp.Connect.{ConnectionsCell, RPConfAsk}
+import coop.rchain.comm.transport.{Blob, TransportLayer}
+import coop.rchain.comm.{transport, PeerNode}
+import coop.rchain.metrics.Metrics
+import coop.rchain.shared.{Log, LogSource, Time}
+import monix.eval.Task
+import monix.execution.Scheduler
+
+import scala.concurrent.duration.FiniteDuration
+import scala.util.Try
+
+trait CasperPacketHandlerInternal[F[_]] {
```

Comment:
If this trait is internal, shouldn't this be enforced by a package private access modifier?

Reply:
It's not internal. Name is confusing (I did not created the name). Wanted to talk / bike-shed about it yesterday - no one was interested... remember? :)